OpenRA.Game/Widgets/Widget.cs

```diff
@@ -128,6 +128,31 @@ public static bool HandleInput(MouseInput mi)
 		/// <param name="e">Key input data</param>
 		public static bool HandleKeyPress(KeyInput e)
 		{
+			if (e.Event == KeyInputEvent.Down)
+			{
+				var hk = Hotkey.FromKeyInput(e);
+
+				if (hk == Game.Settings.Keys.DevReloadChromeKey)
+				{
+					ChromeProvider.Initialize(Game.ModData);
+					return true;
+				}
+
+				if (hk == Game.Settings.Keys.HideUserInterfaceKey)
+				{
+					foreach (var child in Root.Children)
```

Comment:
This can now simplify to `Root.Visible ^= true;` now that the root widget isn't handling the input.